templates/scenarios/csharp/sso-tab/teamsapp.local.yml.tpl

```diff
@@ -3,17 +3,17 @@
 version: 1.0.0
 
 provision:
-  - uses: aadApp/create # Creates a new AAD app to authenticate users if AAD_APP_CLIENT_ID environment variable is empty
+  - uses: aadApp/create # Creates a new AAD app to authenticate users if the environment variable that stores clientId is empty
```

Comment:
@sffamily can you help review update to this line?

The other content has already been reviewed and I have resolved your comments.